CMakeLists.txt

```diff
@@ -4,8 +4,8 @@ project(CedarLogic)
 
 
 set(CEDARLOGIC_MAJOR 2)
-set(CEDARLOGIC_MINOR 3)
-set(CEDARLOGIC_PATCH 5)
+set(CEDARLOGIC_MINOR 4)
+set(CEDARLOGIC_PATCH 2)
```

Comment:
It'd be great if this could auto-version using [semver](https://semver.org/) and `git describe --always` to set the version when it's not directly on a tag. This could be it's own ticket.